commit 637aac2ae3980de31a6baab236a9255fe853cc76
Author: Stefan Liebler <stli@linux.ibm.com>
Date:   Tue Jul 25 11:34:30 2023 +0200

    Include sys/rseq.h in tst-rseq-disable.c
    
    Starting with commit 2c6b4b272e6b4d07303af25709051c3e96288f2d
    "nptl: Unconditionally use a 32-byte rseq area", the testcase
    misc/tst-rseq-disable is UNSUPPORTED as RSEQ_SIG is not defined.
    
    The mentioned commit removes inclusion of sys/rseq.h in nptl/descr.h.
    Thus just include sys/rseq.h in the tst-rseq-disable.c as also done
    in tst-rseq.c and tst-rseq-nptl.c.
    Reviewed-by: Florian Weimer <fweimer@redhat.com>

diff --git a/sysdeps/unix/sysv/linux/tst-rseq-disable.c b/sysdeps/unix/sysv/linux/tst-rseq-disable.c
index 6d73f77e9621da42..964dc728ac5e7b95 100644
--- a/sysdeps/unix/sysv/linux/tst-rseq-disable.c
+++ b/sysdeps/unix/sysv/linux/tst-rseq-disable.c
@@ -22,6 +22,7 @@
 #include <support/xthread.h>
 #include <sysdep.h>
 #include <thread_pointer.h>
+#include <sys/rseq.h>
 #include <unistd.h>
 
 #ifdef RSEQ_SIG
